container-deployment/docker-compose.md

```diff
@@ -78,9 +78,9 @@ Create `fluentd/Dockerfile` with the following content using the Fluentd [offici
 ```text
 # fluentd/Dockerfile
 
-FROM fluent/fluentd:v1.6-debian-1
+FROM fluent/fluentd:v1.12.0-debian-1.0
 USER root
-RUN ["gem", "install", "fluent-plugin-elasticsearch", "--no-document", "--version", "3.5.2"]
+RUN ["gem", "install", "fluent-plugin-elasticsearch", "--no-document", "--version", "5.0.0"]
```

Comment:
fluent-plugin-elasticsearch v5.0.0 contains an initial support for Elasticsearch Data Stream.
This code is not matured.
So, we should use ES plugin v4.3.3 in documantation for now.